src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -3265,12 +3376,32 @@ public static void lengthOfStay(org.hl7.fhir.r4.model.Encounter encounterResourc
       length_of_stay.setCode("m")
                     .setSystem("http://unitsofmeasure.org")
                     .setValue(minutes);
-      
+
       // Once the Duration object is constructed and defined, create and extension and add this information
       String url = losExtensionUrl;
       Extension lengthOfStay = new Extension(url);
       lengthOfStay.setValue(length_of_stay);
       encounterResource.addExtension(lengthOfStay);
     } 
   }
+
+  public static Extension codeEnrichment(String fieldUrl, String codeUrl) {
```

Comment:
Consider adding JavaDoc.

Reply:
Adding this JavaDoc  

/**
   * This function creates an enrichment template with "mainCode" and "enrichedCode" and
   * fills it with the specific URL's regarding the extension in question 
   * 
   * @param fieldUrl  CodeSystem terminology of the extension
   * @param codeUrl   full URL of the field in question, describing its structure     
   * @return enrichment extension
   */